static/templates/settings/account_settings.hbs

```diff
@@ -8,12 +8,12 @@
                 <form class="email-change-form grid">
                     <div class="input-group">
                         <label class="inline-block title">{{t "Email" }}</label>
-                        <button id="change_email_button" type="button" class="button btn-link small rounded inline-block" {{#unless user_can_change_email}}disabled="disabled"{{/unless}}>
-                            {{page_params.delivery_email}}
-                            <i class="fa fa-pencil"></i>
-                        </button>
-                        <i class="tippy-zulip-tooltip fa fa-question-circle change_email_tooltip settings-info-icon" {{#if user_can_change_email}}style="display: none;"{{/if}} data-tippy-content="{{t 'Email address changes are disabled in this organization.' }}">
-                        </i>
+                        <div id="change_email_button_container" {{#unless user_can_change_email}}class="inline-block email_changes_disabled_tooltip"{{else}}class="inline-block"{{/unless}}>
```

Comment:
This can instead be `class="inline-block {{#unless user_can_change_email}}email_changes_disabled_tooltip{{/unless}}"`